src/pages-conditional/what-is-ethereum.js

```diff
@@ -216,101 +219,105 @@ const StyledCallout = styled(Callout)`
 const cards = [
   {
     emoji: ":bank:",
-    title: "Banking for everyone",
-    description:
-      "Not everyone has access to financial services. But all you need to access Ethereum and its lending, borrowing and savings products is an internet connection.",
+    title: <Translation id="whats-ethereum-banking-card" />,
```

Comment:
Did you try running the app? I don't think you can put a component in this field... I suspect that's why the build is broken. You could just put the id here (i.e. `"whats-ethereum-banking-card"`) & then translate using the `formatMessage` function when you pass it into the component.

Reply:
The translation component seems to be working properly when ran locally. Please let me know if there are any issues.